docker-compose/7.1.N-docker-compose.yml

```
@@ -148,7 +148,7 @@ services:
     ports:
       - "5432:5432"
   solr6:
-    image: quay.io/alfresco/search-services:2.0.2.2
+    image: quay.io/alfresco/search-services:2.0.12
```

```suggestion
    image: quay.io/alfresco/search-services:2.0.2.2
```

tests are failing and it's going EOM this month, so not even worth triaging why